src/components/JSONEdit.tsx

```diff
@@ -1,25 +1,28 @@
-import {JSONObject} from '@open-formulieren/types/lib/types';
+import {type JSONObject, type JSONValue} from '@open-formulieren/types/lib/types';
```

Comment:
not saying this is wrong, but why is this `type` addition needed?

Reply:
It's not needed here, I think. That's it. Always explicitly importing types as type declarations makes me not have to think ever, peek into the module I'm importing from and reasoning if it will ever turn into an [edge case](https://devblogs.microsoft.com/typescript/announcing-typescript-3-8-beta/#type-only-imports-exports).

I guess it's also easier on the compiler, but it's mostly easier on my own brain cycles.